docs/setup.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+apt-get update
```

Comment:
Are these commands to add Sphinx?  Are we able to have `sphinx` as a depedency in `requirements-dev.txt`, if that would cover the requirement?

Reply:
I've removed the file,, it is redundant as the commands are now in build_docs.sh.